consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftExecutors.java

```diff
@@ -61,7 +60,8 @@ public synchronized void start() {
     }
     state = State.RUNNING;
     ibftProcessorExecutor = Executors.newSingleThreadExecutor();
-    timerExecutor = newScheduledThreadPool("IbftTimerExecutor", 1, metricsSystem);
+    timerExecutor =
+        MonitoredExecutors.newScheduledThreadPool("IbftTimerExecutor", 1, metricsSystem);
```

Comment:
Thread pool should probably be configurable or perhaps just make it "BftTimerExecutor"

Reply:
renamed